canonical-web-and-design/ubuntu.com
# Remove server-side subscription fetching.

## Done

- Removed server-side fetching of subscription data and passing to the template as the data is now all fetched by the client.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10425.demos.haus/advantage?test_backend=true) and log in.
- You should get the list of subscriptions.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/279.
